pipeline/prepare_eval.py

```diff
@@ -64,6 +64,7 @@
     args.output_path.mkdir(exist_ok=True, parents=True)
 
     conversion_step = f"&& mv /training_folder/model-averaged.nemo /inference_folder/{model_name}"
+    conversion_step += f" && cp /nemo_model/model_config.yaml /inference_folder/"
```

Comment:
just wondering why is this needed?

Reply:
Some part of the eval process where the model was being restored wanted this file in the same directory as the checkpoint, but I can't remember exactly where it was that was erroring out without this...

Reply:
It shouldn't be needed, so let's remove it for now. If you get an error again, please let me know